backend/src/v2/driver/driver.go

```diff
@@ -209,6 +232,10 @@ func Container(ctx context.Context, opts Options, mlmd *metadata.Client, cacheCl
 		return nil, err
 	}
 	glog.Infof("parent DAG: %+v", dag.Execution)
+	dagTasks, err := mlmd.GetExecutionsInDAG(ctx, dag, pipeline)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
It looks like we only need this call to MLMD when we have a `pvc_mount` with `pvc_reference` set via `task_output_parameter`. I feel this may be a pretty rare scenario, which means we should probably put it inside an if-statement.

Reply:
Good point! I moved this part to inside an if-statement for creating PVC.